dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProductionSpecificOwnOperations.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import io.swagger.v3.oas.annotations.media.Schema
+import org.dataland.datalandbackend.model.enums.lksg.ProcurementCategoryType
+import org.dataland.datalandbackend.utils.JsonExampleFormattingConstants
+
+/**
+ * --- API model ---
```

Comment:
I interpreted it as put everything production related into that folder. Also because it is not named `productionSpecific` (as the subcategory) but `production`.

Reply:
Ok we can also leave it as it is.  Just confused me because it's one folder for two subcategories.  But no strong opinion on that.  
Leave it and I will close this.

Reply:
But no thumbs up this time... I see.

Reply:
Can't give no thumbs up since I am crying on my desk now after your response 😢